src/ruby/lib/grpc/grpc.rb

```diff
@@ -16,9 +16,9 @@
   ruby_version_dirname = /(\d+\.\d+)/.match(RUBY_VERSION).to_s
   distrib_lib_dir = File.expand_path(ruby_version_dirname,
                                      File.dirname(__FILE__))
-  if File.directory?(distrib_lib_dir)
+  begin
```

Comment:
The only problem with this change is that if someone downloaded the cross-compiled package but for some reason it fails to load, then that error will be silently `rescue`'d and most likely they will get the second `LoadError` exception, which will have much less useful debug info (since what they most likely want to see is the first `LoadError`).

How about we keep the original logic, but just change `File.directory?` check to:

```
if !Dir.glob("#{distrib_lib_dir}/grpc_c*").empty?
```